.github/workflows/ci.yml

```
@@ -12,8 +12,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-java@v2
+      - uses: actions/checkout@v3.1.0
+      - uses: actions/setup-java@v3.6.0
```

v3 should be fine for these
```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-java@v3
```